partner_communication_switzerland/models/contracts.py

```diff
@@ -223,7 +223,8 @@ def send_daily_communication(self):
             '|', ('correspondent_id.email', '!=', False),
             ('partner_id.email', '!=', False),
             ('state', '=', 'active'),
-            ('type', 'like', 'S')
+            ('type', 'like', 'S'),
+            ('partner_id', '!=0', 1502623) # if partner is not Demaurex
```

Comment:
1. You have a typo: '!=0'
2. Condition field is 'partner_id.ref'
3. 1502623 should be a quoted string
4. pylint issue: you should put 2 spaces before inline comment

Reply:
That was a really bad commit.